src/models/streaming/SessionConfig.ts

```diff
@@ -12,9 +13,11 @@ export class SessionConfig {
      */
     constructor(
         metadata?: string,
-        customVocabularyID?: string
+        customVocabularyID?: string,
+        filterProfanity?: boolean
```

Comment:
It's important to note in this PR that Kosta and I discussed making SessionConfig an interface to be more inline with the async api JobOptions interface object, but making this change would break the versioning of the sdk so we chose to leave SessionConfig as a class.